pkg/fileservice/local_fs.go

```diff
@@ -309,7 +319,20 @@ func (l *LocalFS) Read(ctx context.Context, vector *IOVector) (err error) {
 		}()
 	}
 
-	if err := l.read(ctx, vector); err != nil {
+	// collect read info only when cache missing
+	sCtx, span := trace.Start(ctx, "LocalFS.Read", trace.WithKind(trace.SpanKindLocalFSVis))
```

Comment:
should not introduct sCtx here, just overwrite ctx.

Reply:
fixed